src/Form/EventTypeForm.php

```diff
@@ -413,7 +413,7 @@ public function save(array $form, FormStateInterface $form_state) {
 
     $message = ($status == SAVED_UPDATED) ? '%label event type updated.' : '%label event type added.';
     $url = $event_type->toUrl();
-    $t_args = ['%label' => $event_type->id(), 'link' => Link::fromTextAndUrl(t('Edit'), $url)];
+    $t_args = ['%label' => $event_type->id(), 'link' => (Link::fromTextAndUrl($this->t('Edit'), $url))->toString()];
```

Comment:
Good catch, also it needs separate issue (probably it exists)

Moreover it needs to rework messages as they are not translatable yet

Reply:
https://www.drupal.org/project/rng/issues/3160120

Reply:
Similar https://www.drupal.org/project/rng/issues/3160117#comment-13770476